NEW_PROJECTS.md

```diff
@@ -0,0 +1,70 @@
+# Creating a new SPIFFE Project
+
+We are grateful for your desire to expand the SPIFFE ecosystem with your contribution!
+
+This is an evolving set of guidelines and instructions for how to bring your existing project, or idea for a completely fresh project, under the SPIFFE organization.
+
+If your project is already under the SPIFFE banner, you may be looking for [MATURITY](/MATURITY.md) instead.
+
+## Notice
+
+Maintainer groups are given significant freedom in managing their projects to that we may best foster growth and innovation. There are some elements of SPIFFE subproject ownership to be aware of:
+The [SPIFFE Steering Committee](/ssc/README.md) (SSC) governs the SPIFFE organization as a whole, and has the final authority on acceptance, rejection, or eviction of projects. Additionally, as stewards of community health, the SSC is responsible for intervening in any [Code of Conduct](/CODE-OF-CONDUCT.md) violations or derelictions of maintainer responsibility.
```

Comment:
These responsibility would be easier to follow (on a web-based medium) if they were bullet points.

Reply:
updated